lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -79,7 +79,7 @@ const PolygonParameterEditor = React.createClass({
             points.push(CesiumMath.toDegrees(cartographic.latitude));
             pointsLongLats.push(points);
         }
-        this.props.parameterValues[this.props.parameter.id] = pointsLongLats;
+        this.props.parameterValues[this.props.parameter.id] = [pointsLongLats];
```

Comment:
My two cents on changing props:
* Actually changing a value in props is illegal, I think React will actually stop you from doing that.
* Changing the value of something inside props (e.g. a model-ish object like `parameterValues`) inside `render` or something called by `render` is bad because `render` should idempotent.
* Changing a value of something inside props outside of render is fine, we do that all the time!
* It _may_ be ok to change something inside props even from within a render if the "change" isn't so much a change as it is setting an initial value, e.g. initialize on first access.  Ideally the model layer would do this sort of thing instead of the UI, though.